### PR TITLE
Fix Unitful compat

### PR DESCRIPTION
**Pull request by mjram0s**

### CODE REVIEW

**Review by mattBrzezinski — APPROVED** [review]

**Comment by mjram0s** [comment]
Unitful 1.7 actually breaks the tests so we'll need to restrict to 1.6 for now

File: Project.toml [hunk 1]
```diff
@@ -2,15 +2,15 @@ name = "PowerSystemsUnits"
...
+version = "0.2.1"
 
 [deps]
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [compat]
-Unitful = "0.15, 1.6"
+Unitful = "0.15 - 1.6"
```
Comment (mjram0s) [inline Project.toml:13]: Do we ever use this syntax? I want to keep the compat to 1.6 max but also support past versions of it.

Another option is to do "0.15, ~1.6" but this is more restrictive

Comment (mjram0s) [reply Project.toml:13]: Oh it doesn't work for Julia 1.0

**Comment by mjram0s** [comment]
Resolves #16

**Comment by kmacdough** [comment]
This update does not appear to have been pushed to JuliaHub. Latest version is 0.2.0 (causing Unitful to downgrade).

https://juliahub.com/ui/Packages/PowerSystemsUnits/T4LVl/0.2.0

**Comment by iamed2** [comment]
@kmacdough thanks for pointing that out. Should be addressed soon: https://github.com/JuliaRegistries/General/pull/71862

**Pull request merged**